pkg/errors/errors_test.go

```diff
@@ -23,3 +38,2006 @@ func TestErrorFind(t *testing.T) {
 	Expect(exists).To(Equal(false))
 	Expect(err).To(BeNil())
 }
+
+func Test_NewErrorFromHTTPStatusCode(t *testing.T) {
+	badRequestReason := "bad request reason"
+	unauthorizedReason := "unauthorised reason"
+	forbiddenReason := "forbidden reason"
+	notFoundReason := "not found reason"
+	notImplementedReason := "not implemented reason"
+	conflictReason := "conflict reason"
+	generalReason := "general reason"
```

Comment:
[optional] - we could use some generic string for error message, since we are testing Error types and reuse this generic string message across the tests